cmd/atlas/internal/cmdapi/cmdapi.go

```diff
@@ -287,6 +282,13 @@ func addFlagExclude(set *pflag.FlagSet, target *[]string) {
 
 func addFlagLog(set *pflag.FlagSet, target *string) {
 	set.StringVar(target, flagLog, "", "Go template to use to format the output")
+	// Use MarkHidden instead of MarkDeprecated to avoid
+	// spam users' system logs with deprecation warnings.
+	cobra.CheckErr(set.MarkHidden(flagLog))
+}
+
+func addFlagFormat(set *pflag.FlagSet, target *string) {
+	set.StringVar(target, flagFormat, "", "Go template to use to format the output")
```

Comment:
I think you can merge these two. We can simply remove adding the `--log` in the future. Especially since `log` is just an BC alias for `format`.

Reply:
I don't want to add the `log` option to `schema diff` and `migrate diff`.